crawl-ref/source/dat/strings/de/ui.txt

```diff
@@ -474,7 +474,7 @@ Picks a random sprint map
 Wähle einen zufälligen Dungeon-Sprint-Plan
 %%%%
 Select your previous sprint map and character
-Wähle deine letzte Dungeon-Pan and Charakter
+Wähle deine letzte Sprint-Plan and Charakter
```

Comment:
Wähle deine letzte Sprintkarte und Charakter

Reply:
Will do in separate PR